extensions/publisher-command-center/src/components/ContentsComponent.js

```diff
@@ -72,6 +76,28 @@ const ContentsComponent = {
                 "td",
                 m("button", {
                   class: "action-btn",
+                  ariaLabel: `Rename ${title}`,
+                  title: `Rename ${title}`,
+                  "data-bs-toggle": "modal",
+                  "data-bs-target": `#renameModal-${guid}`,
+                }, [
+                  m("i", { class: "fa-solid fa-pencil" })
+                ]),
+              ),
+              m(
+                "td",
+                m(LockContentButton, {
+                  contentId: guid,
+                  contentTitle: title,
+                  isLocked: content["locked"],
```

Comment:
Oh sweet we have `isLocked` for the component, we can use that in `Contents.lock` and send it up to the API (or have two endpoints)